flexmeasures/api/v3_0/tests/test_sensor_data.py

```diff
@@ -213,3 +224,45 @@ def test_post_sensor_data_twice(client, setup_api_test_data, requesting_user):
     print(response.json)
     assert response.status_code == 403
     assert "data represents a replacement" in response.json["message"]
+
+    # at this point, the transaction has failed and needs to be rolled back.
+    db.session.rollback()
+
+
+@pytest.mark.parametrize(
+    "num_values, status_code, message, saved_rows",
+    [
+        (1, 200, "Request has been processed.", 1),
+        (2, 500, "Cannot save multiple instantaneous values simultaneously.", 0),
```

Comment:
Potential discrepancy:
- What this suggests is that you cannot save, in a single API call, multiple instantaneous values.
- What I believe this actually tests is that you cannot save multiple instantaneous values that overlap. That is, two values spanning the same moment (a zero duration).